batch/src/main/java/org/frankframework/batch/RecordTransformer.java

```diff
@@ -664,6 +664,39 @@ public IOutputField appendValue(IOutputField curFunction, StringBuilder result,
 		}
 	}
 
+	/*
```

Comment:
Heeft Apache Commons hier niet al wat utility methods voor?

Reply:
Geen idee, maar dit zit nu in `RecordTransformer` met: `@deprecated Warning: non-maintained functionality`.
Dus wil je dat we dat gaan fixen?

Reply:
Aangepast naar Commons inderdaad, met een klein piefje er bij.

Reply:
Die andere method heb ik niet aangepast @tnleeuw, omdat ik daar geen coverage op heb en die niet letterlijk te mappen is.